wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue

```diff
@@ -40,18 +40,45 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <v-layout
       row
       wrap
-      class="border-box-sizing mx-0">
+      class="border-box-sizing mx-0 infoMenu ">
       <v-spacer />
       <v-flex
-        md3
-        offset-xs0
-        xs12
-        class="mt-2 mx-4 border-box-sizing">
+        class="mt-2 mx-4 border-box-sizing walletTextField">
         <v-text-field
           v-model="search"
           :label="$t('exoplatform.wallet.label.searchInWalletPlaceholder')"
           prepend-inner-icon="fa-filter"
-          class="pt-0 mt-0 walletTextField" />
+          class="pt-0 mt-0" />
+      </v-flex>
+      <v-flex
+        class="mt-2 mx-4 border-box-sizing dropDownFilter">
+        <v-menu
+          v-model="showMenu"
+          offset-y>
+          <template v-slot:activator="{ on }">
+            <button
+              class="btn"
+              v-on="on"
+              @blur="closeMenu">
+              {{ newsFilterLabel }}
+              <i class="uiIconMiniArrowDown uiIconLightGray"></i>
+            </button>
+          </template>
+          <v-list>
+            <v-list-item @mousedown="$event.preventDefault()">
+              <v-list-item-title class="filterLabel" @click="filterWallet ='all'">{{ $t('exoplatform.wallet.label.All') }}</v-list-item-title>
+            </v-list-item>
+            <v-list-item @mousedown="$event.preventDefault()">
+              <v-list-item-title class="filterLabel" @click="filterWallet ='Disabled wallets by admin'">{{ $t('exoplatform.wallet.label.disabledWalletsByAdmin') }}</v-list-item-title>
+            </v-list-item>
+            <v-list-item @mousedown="$event.preventDefault()">
+              <v-list-item-title class="filterLabel" @click="filterWallet ='Wallets of disabled users'"> {{ $t('exoplatform.wallet.label.disabledUsersWallets') }}  </v-list-item-title>
+            </v-list-item>
+            <v-list-item @mousedown="$event.preventDefault()">
+              <v-list-item-title class="filterLabel" @click="filterWallet ='Rejected wallets'">{{ $t('exoplatform.wallet.label.rejectedWallets') }}</v-list-item-title>
```

Comment:
Could you use shorter text for filters with no spaces. it will be less error prone.

Reply:
done!